front/src/modules/ui/object/record-table/components/RecordTableHeaderPlusButtonContent.tsx

```diff
@@ -28,19 +30,32 @@ export const RecordTableHeaderPlusButtonContent = () => {
     [handleColumnVisibilityChange, closeDropdown],
   );
 
+  const iconButtons = (column: ColumnDefinition<FieldMetadata>) => {
+    return [
+      {
+        Icon: IconPlus,
+        onClick: () => handleAddColumn(column),
+      },
+    ];
+  };
+
   return (
     <DropdownMenuItemsContainer>
       {hiddenTableColumns.map((column) => (
         <MenuItem
           key={column.fieldMetadataId}
-          iconButtons={[
-            {
-              Icon: IconPlus,
-              onClick: () => handleAddColumn(column),
-            },
-          ]}
+          iconButtons={
+            column.fieldMetadataId !== 'customizeFields'
```

Comment:
Let's avoid this and hard-code the option in its own MenuItem below this hiddenTableColumns.map